### ISSUE TITLE
I2 P4a

### ISSUE THREAD

**Issue by pxparga** [PUC-IIC2283/2020-IIC2283-Pruebas#18]
Hola! en la P4a la solución tiene que ser determinista? o puede ser como los algoritmos aleatorizados que vimos

**Comment by marceloarenassaavedra** [comment]
El algoritmo va a ser aleatorizado porque tiene que usar a la función LanzarMoneda() que es aleatoria.

Saludos!

**Comment by pxparga** [comment]
si, pero me refiero a resultado. si es que este cumple con estar entre 1 y p-1

**Comment by marceloarenassaavedra** [comment]
Sí, el resultado es un numero aleatorio en el conjunto {1, ..., p-1}
